src/test/java/StringTest.java

```diff
@@ -0,0 +1,56 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class StringTest {
+    @Test
+    void splitTwoNumbersTest() {
+        String target = "1,2";
+        String delimiter = ",";
+        
+        String[] result = target.split(delimiter);
```

Comment:
```suggestion
        String[] result = "1,2".split(",");
```
로컬 변수 없이 구현해도 되지 않을까?